varnish-sys/build.rs

```diff
@@ -19,6 +19,7 @@ impl VarnishInfo {
     fn parse(bindings: PathBuf, varnish_paths: Vec<PathBuf>, version: String) -> Self {
         let (major, minor) = parse_version(&version);
 
+        // >= 7.0 .. < 7.6
```

Comment:
this is not what the `if` statement says (version 8 is coming)

Reply:
???  the current `if` statement says exactly that, unless i can't read code (which is possible...)

Reply:
you are correct, **I** can't read code